### PR TITLE
Fix Material Design Icons when CDN is disabled.

### LINKED ISSUE #9520: admin menu on local intranet wait for fonts.googleapis.com ...
I run Orchardcore on my local intranet and when I change some setting in the admin menu, it's waiting for `looking up for fonts.googleapis.com`
I disabled UseCDN :
` public void ConfigureServices(IServiceCollection services)
        {
            services.AddOrchardCms().AddMvc().ConfigureServices(tenantServices => {
                tenantServices.PostConfigure<ResourceManagementOptions>(o => o.UseCdn = false);
            });
        }`
but nothing happened.

### PR DESCRIPTION
**Pull request by Skrypt**
Fixes #9520

### CODE REVIEW

File: src/OrchardCore.Themes/TheAdmin/wwwroot/fonts/material-design-icons/material-icons.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@font-face {
+  font-family: 'Material Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: url(MaterialIcons-Regular.eot); /* For IE6-8 */
+  src: local('Material Icons'),
+       local('MaterialIcons-Regular'),
+       url(MaterialIcons-Regular.woff2) format('woff2'),
+       url(MaterialIcons-Regular.woff) format('woff'),
+       url(MaterialIcons-Regular.ttf) format('truetype');
```
Comment (deanmarcussen) [inline src/OrchardCore.Themes/TheAdmin/wwwroot/fonts/material-design-icons/material-icons.css:15]: Looks great.

Does it include all the icons? Here we have outlined, and twotone etc
`.SetCdn("https://fonts.googleapis.com/icon?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp")`

and this is just Regular? But I don't know what's in regular, so just checking ;)

Comment (Skrypt) [reply src/OrchardCore.Themes/TheAdmin/wwwroot/fonts/material-design-icons/material-icons.css]: Ok made some changes to support all icon styles.

**Review by deanmarcussen — APPROVED** [review]
👍

**Comment by ns8482e** [comment]
@Skrypt @deanmarcussen After taking latest with this changes, I am getting an error 

InvalidOperationException: Could not find a resource of type 'script' named 'the-admin' with version '1'.

**Comment by Skrypt** [comment]
I don't get this issue with source code on latest dev branch.

**Comment by ns8482e** [comment]
It was build issue while running with power shell

**Pull request merged**